crowdin_api/api_resources/screenshots/resource.py

```diff
@@ -34,6 +34,8 @@ def list_screenshots(
         page: Optional[int] = None,
         offset: Optional[int] = None,
         limit: Optional[int] = None,
+        labelIds: Optional[Iterable[int]] = None,
+        excludeLabelIds: Optional[Iterable[int]] = None
```

Comment:
please also add the `stringId` parameter here and put the new parameters after the `projectId`

Reply:
the labelIds and excludeLabelIds also put after the stringId